backend/models/game.go

```diff
@@ -3,6 +3,7 @@ package models
 import "time"
 
 type Game struct {
+	Id           int
```

Comment:
The id field is redundant. You can use db.GetGameById instead

Reply:
Nevermind #53

Reply:
Sorry to blow this up, but make the ID a string because mongo uses hex IDs and firestore uses ints, so a string would work best.

Reply:
Okay I'll get that done this afternoon and merge